slideruler-info.patch

```diff
@@ -0,0 +1,31 @@
+diff --git a/activity/activity.info b/activity/activity.info
+index 076b7f1..aac09d4 100644
+--- a/activity/activity.info
++++ b/activity/activity.info
+@@ -1,10 +1,19 @@
+ [Activity]
+-name = Sliderule
++name = Slideruler
+ activity_version = 35
+-license = GPLv3
+-bundle_id = org.sugarlabs.SlideruleActivity
++license = GPLv3+
++bundle_id = org.sugarlabs.Sliderule
+ exec = sugar-activity3 SlideruleActivity.SlideruleActivity
+ icon = activity-sliderule
+ show_launcher = yes
+-summary = a mechanical calculator for multiplication, division, log, and more
+-category = maths
+\ No newline at end of file
++summary = A mechanical calculator
++category = maths
++release_date = 2019-12-28
++metadata_license = CC0-1.0
++repository = https://github.com/sugarlabs/slideruler
++url = https://wiki.sugarlabs.org/go/Activities/Sliderule
++description = The slide rule also known colloquially as a slipstick, is a mechanical analog computer. This Activity implements a basic slide rule with C, D, A, and L scales and can be used for multiplication, division, roots, addition and subtraction. 
++update_contact = tch@sugarlabs.org
++developer_name = Sugar Labs Community
++screenshots = https://i.imgur.com/qLWnoaW.png
++tags = Game
```

Comment:
It doesn't look like a game, better use "Education", which is the same I used for Abacus.